tests/testValueProviders.py

```diff
@@ -235,8 +243,13 @@ def checkOutput():
         for t in threads:
             t.join()
 
+
+        numpy.testing.assert_equal(opCompute.executionCount, 1)
+        assert opCache._dirty == False
```

Comment:
And this assert also fails occasionally, because of line 232.  I think the assert is valid if we request output from the cache one last time.  Will push a fix in a minute.